seabolt/src/connect.c

```diff
@@ -544,7 +538,9 @@ void BoltAddress_resolve_b(struct BoltAddress * address)
                 case AF_INET:
                 case AF_INET6:
                 {
-                    _copy_socket_address(BoltAddress_resolved_host(address, p), (struct sockaddr_storage*)(ai_node->ai_addr));
+                    struct sockaddr_storage * target = BoltAddress_resolved_host(address, p);
+                    struct sockaddr_storage * source = (struct sockaddr_storage*)(ai_node->ai_addr);
```

Comment:
I think we don't need this cast anymore. Passing it as `ai_node->ai_addr` would be perfectly fine.

Reply:
If I drop the cast, it complains about incompatible pointer types.